docs/virtual_machines/templates.md

```diff
@@ -6,15 +6,15 @@ KubeVirt on OpenShift).
 
 By deploying KubeVirt on top of OpenShift the user can benefit from the
 [OpenShift
-Template](https://docs.openshift.org/latest/dev_guide/templates.html)
+Template](https://docs.openshift.com/container-platform/4.6/virt/vm_templates/virt-creating-vm-template.html)
 functionality.
 
 ## Virtual machine templates
 
 ### What is a virtual machine template?
 
 The KubeVirt projects provides a set of
-[templates](https://docs.okd.io/latest/dev_guide/templates.html) to
+[templates](https://docs.okd.io/latest/openshift_images/using-templates.html) to
```

Comment:
Do we want to have this on latest and risk that it gets out of date again or do we pin it to a specific version?

Reply:
Templates are only good in Red Hat OpenShift. I searched the OKD latest documentation and it seems the Red Hat OKD links in this page refer to general template use not KubeVirt virtual machine common templates. 

I recommend we change the Red Hat OKD links to Red Hat OpenShift links.

Reply:
Understood. Thanks for checking